src/main/drivers/timer.h

```diff
@@ -107,6 +111,26 @@ typedef struct timerHardware_s {
 #endif
 } timerHardware_t;
 
+#ifdef USE_TIMER_MGMT
+
+typedef uint8_t timerTag_t;
+
+typedef struct timerChannelConfig_s {
```

Comment:
Please note it is also there for backward compatibility. We can enhance over time - taking an iterative approach rather than a big bang.